continuous_integration/appveyor/install.ps1

```diff
@@ -1,16 +1,25 @@
 # Sample script to install Python and pip under Windows
-# Authors: Olivier Grisel and Kyle Kastner
-# License: BSD 3 clause
+# Authors: Olivier Grisel, Jonathan Helmus, Kyle Kastner, and Alex Willmer
+# License: CC0 1.0 Universal: http://creativecommons.org/publicdomain/zero/1.0/
```

Comment:
`ogrisel/python-appveyor-demo`: I did a copy and paste from the master branch of that project because it include many fixes over the old version we had in sklearn.